docs/package.json

```diff
@@ -10,12 +10,13 @@
     "files": [],
     "scripts": {
         "images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
-        "watch:sass": "sass --watch --poll src/assets/scss:src/assets/css",
+        "watch:sass": "sass --watch --poll src/assets/scss:_site/assets/css",
```

Comment:
Could potentially undo this, and then use the src/assets/css as the watch target for postcss

Reply:
We can have it like you have written. This is better. We can reduce writing and copying action to just directly writing to `_site`. Just confirm if changes work fine in dev env